sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -910,7 +910,7 @@ case class ShowCreateTableCommand(table: TableIdentifier) extends RunnableComman
       }
     }
 
-    if (metadata.tableType == EXTERNAL) {
+    if (metadata.tableType == EXTERNAL || metadata.tableType == MANAGED) {
```

Comment:
> SHOW CREATE TABLE shows the CREATE TABLE statement that creates a given table, or the CREATE VIEW statement that creates a given view.

If we add the location clause for managed tables, CREATE TABLE will create an external table instead of a managed table. That is not what we want.